internal/test/smoke/rbac_permission_test.go

```diff
@@ -239,13 +239,14 @@ func (s *SuiteRbacPermission) TestReadPermission() {
 			Name:     "Test idmsvc:domain:create",
 			Given:    s.prepareDomainIpaCreate,
 			Then:     s.doTestDomainIpaCreate,
-			Expected: http.StatusUnauthorized,
+			Expected: http.StatusCreated,
```

Comment:
Is it correct? When user have only read permissions that doing API request to create domain should IMO fail.  It can be different for IPA server doing the request though - a system account.

Reply:
Thanks! It is wrong how I structured the table; but the expected is right; that endpoint would not be reached out by a user; it is enforced the type of XRHID for this endpoint to be `System`. :+1: